packages/react/README.md

```diff
@@ -16,8 +16,6 @@ TODO: add alpha, beta and production statuses to components. Also show in Storyb
 
 ## Getting started
 
-TODO: These packages haven't been published yet, so the following doesn't work at the moment. You can currently only use these instructions when locally linking the packages to a consumer project.
-
 Install the packages you need, for instance:
 
 `npm install @amsterdam/design-system-react @amsterdam/design-system-css @amsterdam/design-system-tokens @amsterdam/design-system-font`
```

Comment:
Do they have to install tokens separately? I would've thought the tokens are compiled to css in the css package.